src/amo/css/OverallRating.scss

```diff
@@ -0,0 +1,40 @@
+.OverallRating {
+
+  form {
+    display: flex;
+    justify-content: center;
+
+    input {
+      appearance: none;
+      opacity: 0;
+      width: 0;
+      height: 0;
+    }
+
+    label {
```

Comment:
As for the verbosity in the HTML you could make a stateless component and shorten the HTML instead:

```jsx
const OverallRatingChoice = ({ children, value }) => (
  <div>
    <input id="love-it" value="love-it" name="overall-rating"
           className={`OverallRating-label-${value}`} type="radio" />
    <label htmlFor="love-it" className="love-it">{children}</label>
  </div>
);

<OverallRatingChoice value="love-it">{i18n.gettext('Love it!')}</OverallRatingChoice>
<OverallRatingChoice value="it-is-okay">{i18n.gettext("It's okay")}</OverallRatingChoice>
<OverallRatingChoice value="huh">{i18n.gettext('Huh?')}</OverallRatingChoice>